README.md

```diff
@@ -645,10 +645,10 @@ We have created a few langague specific packages to help you interact with the p
 
 To cut a [release](https://github.com/cipherstash/encrypt-query-language/releases) of EQL:
 
-1. Draft a [new release](https://github.com/cipherstash/encrypt-query-language/releases/new) on GitHub
-1. Choose a tag, and create a new one with the prefix `eql-` followed by a [semver](https://semver.org/) (for example, `eql-1.2.3`)
-1. Generate the release notes
-1. Optionally set the release to be the latest (you can set a release to be latest later on if you are testing out a release first)
-1. Click the `Publish release` button
+1. Draft a [new release](https://github.com/cipherstash/encrypt-query-language/releases/new) on GitHub.
+1. Choose a tag, and create a new one with the prefix `eql-` followed by a [semver](https://semver.org/) (for example, `eql-1.2.3`).
+1. Generate the release notes.
+1. Optionally set the release to be the latest (you can set a release to be latest later on if you are testing out a release first).
+1. Click `Publish release`.
 
-This will trigger a run of the [Release EQL](https://github.com/cipherstash/encrypt-query-language/actions/workflows/release-eql.yml) workflow, which will build and attach artifacts to the release.
+This will trigger the [Release EQL](https://github.com/cipherstash/encrypt-query-language/actions/workflows/release-eql.yml) workflow, which will build and attach artifacts to the release.
```

Comment:
Link to where the release will show up

```suggestion
This will trigger the [Release EQL](https://github.com/cipherstash/encrypt-query-language/actions/workflows/release-eql.yml) workflow, which will build and attach artifacts to [the release](https://github.com/cipherstash/encrypt-query-language/releases/).
```